Automattic/wp-calypso
# languages: Update languages-meta.json to reflect Norwegian locale changes on WP.com

#### Changes proposed in this Pull Request

* Updates the languages-meta.json in `@automattic/languages` to reflect the locale changes in D76578.
* The updated languages-meta.json has the `revisions` property removed from the entries, since it's no longer used.

#### Testing instructions

* Open Calypso.live build
* Go to Account Settings and open the language switcher
* Confirm `Norsk bokmål` option is available.

Related to 394-gh-Automattic/i18n-issues
